lib/rspec/rails/example/system_example_group.rb

```diff
@@ -75,6 +75,10 @@ def app
 
         attr_reader :driver
 
+        if ::Rails.version.to_f == 5.1
```

Comment:
Could you explain why?

Reply:
The first implementation is lack of considering of compatibility. `action_dispatch_system_test_case` is available from Rails 5.2 . The change is needed to pass `System specs driven by selenium_chrome_headless` spec in Rails 5.1 .

Reply:
It was literally checking the method exists, then using it, can you explain the "lack of compatibility" I'm afraid I don't understand how "not calling the method if it doesn't exist" changes anything... Also note this need to work on 5.2 and beyond, so this check seems incorrect.